addons/web/static/src/views/view_dialogs/export_data_dialog.js

```diff
@@ -71,17 +72,22 @@ export class ExportDataDialog extends Component {
             search: [],
             selectedFormat: 0,
             templateId: null,
+            isSmall: this.env.isSmall,
         });
 
         this.title = this.env._t("Export Data");
         this.newTemplateText = this.env._t("New template");
         this.removeFieldText = this.env._t("Remove field");
         this.expandText = this.env._t("Show sub-fields");
 
+        this.debouncedOnResize = useDebounced(this.onResize, 300);
+        window.addEventListener("resize", this.debouncedOnResize);
```

Comment:
I think you should add the event listener in `mounted`, because as it stands, you could get in a situation where you bound the listener (the component has been setup), but you never unbind it (the component hasn't been mounted because another rendering cancelled this one, so it will never be unmounted). To ensure you have the correct size, I would do
```js
onMounted(() => {
    browser.addEventListener("resize", this.debouncedOnResize);
    this.onResize();
});
```
If the size didn't change, calling onResize will have no impact. Maybe rename `onResize` into `updateIsSmall` or `updateSize` ?

Also, prefer using `browser` instead of window, it eases monkey patching in tests. (same for remove)